src/components/utils/include/utils/timer.h

```diff
@@ -44,9 +44,12 @@ typedef sigval_t sigval;
 
 #include <time.h>
 #include <string>
+#include <limits>
 
 #include "utils/lock.h"
 #include "utils/timer_task.h"
+#include "utils/threads/thread.h"
+#include "utils/threads/thread_delegate.h"
 
 namespace timer {
 typedef uint32_t Milliseconds;
```

Comment:
@anosach-luxoft , you not enough timer for  49,71 days you want timer for 584942,41 millenniums?!